### ISSUE TITLE
Decision Trees

### ISSUE THREAD

**Issue by dianzrong** [uoft-tmi/signature-forgery-detection#6]
### Purpose
It's important to try simpler models like Decision Trees before more intricate models to ensure we don't overly complicate our solution.

### Task
- [x] Understand, experiment with, and implement the structure for the Decision Tree model
- [x] Update the model to take in the processed images and return whether the signature is a forgery
- [x] Train the model on the train dataset
- [x] Fine-tune the model (depth of tree, way to measure the split, using multiple trees, etc.) on the validation dataset
- [x] Report the model's accuracy on the test set

### Resources

- [Theory and code for Decision Trees](https://www.geeksforgeeks.org/decision-tree/)
- [Possible parameters to fine-tune](https://towardsdatascience.com/how-to-tune-a-decision-tree-f03721801680)
